sway-parse/src/expr/mod.rs

```diff
@@ -0,0 +1,1145 @@
+use crate::priv_prelude::*;
```

Comment:
I love the idea of a private prelude. We should use this more!

Reply:
Yeah I write all my crates like this. Other than making it easier to manage imports it also forces things to have a consistent meaning throughout the crate (eg. so `Span` doesn't mean two different things in two different files).